content/linked-terms.js

```diff
@@ -0,0 +1,14 @@
+module.exports = [
```

Comment:
Could this be extracted to a simple JSON or YAML config file so we can easily edit the list later? I mean... It's already easy but would be even easier 🙂

Also, should a test be written for this fn?

Reply:
It could, but using `.js` allows us to use RegExp literal matchers and dynamic replacement values in a future. It's possible to switch to JSON by simply converting what's there now if we decide we don't want those features.